ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -85,6 +85,7 @@ spec:
       optional: true
     - name: registry-pull-credentials
     - name: registry-push-credentials
+    - name: registry-read-credentials
```

Comment:
`read` and `pull` are kind of synonymous for a registry. How about using `serve` instead (i.e. these creds are only used to serve the released images)?